content/en/docs/tasks/manage-kubernetes-objects/kustomization.md

```diff
@@ -808,14 +808,14 @@ deployment.apps "dev-my-nginx" deleted
 | commonLabels          | map[string]string                                                                                            | labels to add to all resources and selectors                                       |
 | commonAnnotations     | map[string]string                                                                                            | annotations to add to all resources                                                |
 | resources             | []string                                                                                                     | each entry in this list must resolve to an existing resource configuration file    |
-| configmapGenerator    | [][ConfigMapArgs](https://github.com/kubernetes-sigs/kustomize/blob/master/pkg/types/kustomization.go#L195)  | Each entry in this list generates a ConfigMap                                      |
-| secretGenerator       | [][SecretArgs](https://github.com/kubernetes-sigs/kustomize/blob/master/pkg/types/kustomization.go#L201)     | Each entry in this list generates a Secret                                         |
-| generatorOptions      | [GeneratorOptions](https://github.com/kubernetes-sigs/kustomize/blob/master/pkg/types/kustomization.go#L239) | Modify behaviors of all ConfigMap and Secret generator                             |
+| configmapGenerator    | [][ConfigMapArgs](https://github.com/kubernetes-sigs/kustomize/blob/master/api/types/kustomization.go#L99)  | Each entry in this list generates a ConfigMap                                      |
```

Comment:
To be more specific, please consider changing the link to something like
`https://github.com/kubernetes-sigs/kustomize/blob/release-kustomize-v4.0/api/types/kustomization.go#L99`
where the `master` is replaced by a tag `release-kustomize-v4.0`.
The revised link will not be a moving target then.

Reply:
@npu21 Maybe you can consider the view from tengqm .
Some files are archived and we should consider linking to them.
Thanks!